packages/date-picker/src/DatePicker/DatePickerInput/DatePickerInput.tsx

```diff
@@ -111,7 +111,7 @@ export const DatePickerInput = forwardRef<HTMLDivElement, DatePickerInputProps>(
 
       switch (key) {
         case keyMap.ArrowLeft: {
-          // Prevent the cursor on tab and initial click
+          // Without this, the input does not select all the text
```

Comment:
I'm still a bit unclear on how preventing default keydown behavior prevents selection

Reply:
I'm unsure how to explain this -- without preventDefault, the input ignores `.select()`, and I'm unsure why.

Reply:
Hmm, If you have time, this would be an interesting thing to investigate why